web/app/themes/clarity/inc/admin/prior-party/prior-party-banner-admin.php

```diff
@@ -212,6 +212,19 @@ public function page(): void
             $stop = new \DateTime($this->banner["end_date"]);
 
 
+            // Init an array to hold the query string for viewing the banner.
+            $link_view_queries = [];
+            // If the banner is inactive, we need to set a query string to show it.
```

Comment:
I'm not sure this is true @EarthlingDavey - we can see inactive banners on the front end using the `time_context` - what is `show_inactive` used for? Are we changing logic?

Reply:
Also, `show_inactive` doesn't mean anything to an end user that would see the URL. 
If you are using this in replacement of `time_context` would a name like `preview_prior` be more understandable?

Reply:
The thing is, I think it was a bug that we set `time_context ` on every banner link.

It was meant to be a 'time travel forward' past the banner's end date. If the current time is past the banner's end date then time_context isn't really needed.

The logic is pretty much the same, but instead of inferring from `time_context` that we should load inactive banners, there's a query string specifically for that (`show_inactive ` just changed to `preview_unpublished`).

I think we're better off doing it like that otherwise we could run into a confusing bug where a dev assumes that time_context is tightly linked to showing inactive banners, and not realise that the behaviour is inferred.